src/components/material/ReviewInfomedia.tsx

```diff
@@ -80,7 +80,7 @@ const ReviewInfomedia: React.FC<ReviewInfomediaProps> = ({
           ) : (
             t("cantViewReviewText")
           )}
-        </div>
+        </h4>
```

Comment:
Please reconsider using a `<h4>` here.

As I see it this is not an actual headline. It is a button to show that the user cannot see the review without logging in.